docs/v1/openapi.yaml

```diff
@@ -503,6 +522,29 @@ paths:
                 - code
                 - code_verifier
         description: ''
+  /oidc/callback:
+    get:
+      summary: OIDC Callback
+      tags:
+        - oidc4vc
+      operationId: oidc-callback
+      description: OIDC callback for storing external issuer auth code and continue our oidc auth flow.
```

Comment:
... for handling response from issuer's OIDC provider and continue our OIDC authorize flow.

Reply:
done